src/domain/notifications/notifications.repository.v2.ts

```diff
@@ -144,7 +139,11 @@ export class NotificationsRepositoryV2 implements INotificationsRepositoryV2 {
     chainId: string;
     safeAddress: `0x${string}`;
   }): Promise<void> {
-    return this.notificationsDatasource.deleteSubscription(args);
+    return this.notificationsDatasource.deleteSubscription({
+      deviceUuid: args.deviceUuid,
+      chainId: args.chainId,
+      safeAddress: args.safeAddress,
+    });
```

Comment:
Another small nit, but this could remain as `this.notificationsDatasource.deleteSubscription(args)`, couldn't it?

Reply:
Changed in da9bf557fe0afb1b1eabea3681f878250e7c90ee.